hadoop-hdds/managed-rocksdb/src/main/java/org/apache/hadoop/hdds/utils/db/managed/JniLibNamePropertyWriter.java

```diff
@@ -39,8 +39,8 @@ public static void main(String[] args) {
     String filePath = args[0];
     try (Writer writer = new OutputStreamWriter(
         Files.newOutputStream(Paths.get(filePath)), StandardCharsets.UTF_8)) {
-      writer.write("rocksdbLibName=" +
-          Environment.getJniLibraryFileName("rocksdb"));
+      String libName = Environment.getJniLibraryFileName("rocksdb");
```

Comment:
Why not use `public static String getRocksDBLibFileName() {`

Reply:
done